CHANGELOG.md

```diff
@@ -5,6 +5,42 @@ All notable changes for each version of this project will be documented in this
 ## 12.1.0
 
 ### New Features
+- Added `IgxAccordion` component
+    - A collection of vertically collapsible igx-expansion panels that provide users with data and the ability to drill into it in a compact manner. The control is **not** data bound and takes a declarative approach, giving users more control over what is being rendered.
+    - Exposed two expansion modes, templating, easy-to-use keyboard navigation, option for nested accordions and more.
```

Comment:
Even though I quite like the `and more` ending, this document is not related to marketing. So why don't we include everything else that a potential user will want to know about the accordion? One thing that comes to mind is the fact that the user can say whether they want single branch expanding or not. So instead of saying **expansion modes** we can simply state them.

Also, I don't think we need to say **templating** because the accordion is a wrapper around the expansion panels.